src/System.Private.CoreLib/shared/System/Globalization/CalendarData.Unix.cs

```diff
@@ -39,7 +39,11 @@ private bool LoadCalendarDataFromSystem(string localeName, CalendarId calendarId
             // TODO-NULLABLE: these can return null but are later replaced with String.Empty or other non-nullable value
             result &= GetCalendarInfo(localeName, calendarId, CalendarDataType.NativeName, out this.sNativeName!);
             result &= GetCalendarInfo(localeName, calendarId, CalendarDataType.MonthDay, out this.sMonthDay!);
-            this.sMonthDay = NormalizeDatePattern(this.sMonthDay);
+
+            if (this.sMonthDay != null)
+            {
+                this.sMonthDay = NormalizeDatePattern(this.sMonthDay);
+            }
```

Comment:
Is this the code path that could be affected by missing resources?

Reply:
resources here doesn't mean the localized resources, it means the ICU resources (e.g. day or month names...etc.). So, this is the code path we read/get this ICU resources.

Reply:
Understood.  But this line is just accounting for sMonthDay not having been read.  I'm asking if there are others besides that, where we assume some field was filled in but it wasn't because ICU was missing the relevant resource.

Reply:
The method LoadCalendarDataFromSystem is the place we fill such properties. sMonthDay is the only property we read and access it before we check it. Note the rest of the code in this method doesn't try to access any of such properties we read. if you look at the other file CultureData.cs where I removed the assert, this is the palace we check which property we couldn't read and then fill it with the invariant values.

Reply:
Ok, thanks.